jetson/src/core/robot.cpp

```diff
@@ -1,11 +1,14 @@
 #include "robot.h"
 
+#include "log.h"
+#include "cluster.h"
+
 #include <cmath>
 
 #define IDEAL_LINEAR_SPEED 0.1
 
-Robot::Robot(double robotLength, double robotWidth, double wheelRadius, Camera & camera, double robotPosTol, double armPosTol) 
-    : robotLength(robotLength), robotWidth(robotWidth), wheelRadius(wheelRadius), camera(camera), robotPosTol(robotPosTol), armPosTol(armPosTol) {
+Robot::Robot(double robotLength, double robotWidth, double cameraHeight, double wheelRadius, Camera & camera, double robotPosTol, double armPosTol) 
+    : robotLength(robotLength), robotWidth(robotWidth), cameraHeight(cameraHeight), wheelRadius(wheelRadius), camera(camera), robotPosTol(robotPosTol), armPosTol(armPosTol) {
```

Comment:
All these should just be defined as constants but can do that later. We only have 1 robot type.